styles/disktop.css

```diff
@@ -183,4 +183,46 @@
   .with-border {
     padding-bottom: 24px;
   }
+
+  /* Contact section */
+
+  .relative-pos {
+    position: relative;
+  }
+
+  .contact-sec {
+    display: flex;
+    align-items: center;
+    max-height: 846px;
+    padding: 0;
+    justify-content: center;
+    border-top-left-radius: 150px;
+    background-image: url(../images/contact\ form\ background\ shapes.svg);
+    background-position: 50% 100%;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 700px;
+  }
+
+  .contact-form {
+    max-width: 801px;
+  }
```

Comment:
- You did a great job when designing your desktop version however there is some small issue you still need to fix in the contact form.  I list them below please fix them it will make your page stick to the Figma design as required is the project :+1: :grinning: :grinning: :grinning: 
  - Kindly make sure you have a background image  stick to the one shown in the Figma design 
  - Kindly make the background color blue as shown in the Figma design 
  - The content of your page must be centered in the middle of the page and make sure the disign you add is not affect the mobile version I suggest you make your max-width 100% instead of 801px;


## Your contact form  design 

![image](https://user-images.githubusercontent.com/95297251/207369954-06e6da5f-ec75-4af3-aa79-827aa66903d2.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/207370165-89d611c2-63b3-493d-8bd4-7469cd48e702.png)

Reply:
Thank you for your valuable feedback.
I have checked it. It is fine. I re-created it using the figma design.